src/package/nuspec/Microsoft.NET.Test.Sdk.nuspec

```diff
@@ -17,9 +17,10 @@
     </dependencies>
   </metadata>
   <files>
+    <file src="Microsoft.NET.Test.Sdk.props" target="buildMultiTargeting\" />
```

Comment:
You shouldn't need to. NuGet will still import and execute the targets file on the "inner" build. It looks like the current targets file only generates a "Program.cs" file, and which depends on `TargetFrameworkIdentifier` being set. It probably will work incorrectly when running in the "outer" build.

Btw, if you're not familiar with inner vs outer, here is _conceptually_ what MSBuild is doing in multi-targeting projects:
```xml
<!-- this executes when `msbuild.exe /t:Build` runs on the project -->
<Target Name="Build" Condition="'$(TargetFrameworks)' != '' AND '$(TargetFramework)' == ''">
    <!-- import buildMultiTargeting/Microsoft.NET.Test.Sdk.props  -->
    <ItemGroup>
      <_TargetFramework Include="$(TargetFrameworks)" />
    </ItemGroup>
    <!-- starts a new "build" of the project, once per target framework -->
    <MSBuild Projects="$(MSBuildProjectFile)"
             Targets="InnerBuild"
             Properties="TargetFramework=%(_TargetFramework.Identity)" />
</Target>

<Target Name="InnerBuild" Condition="'$(TargetFramework)' != ''">
   <!-- import build/$(TargetFramework)/Microsoft.NET.Test.Sdk.{props, targets} -->
   <!-- gather *.cs files, run compiler, etc. -->
</Target>
```